gitlab/provider.go

```diff
@@ -93,9 +94,13 @@ func Provider() terraform.ResourceProvider {
 			"gitlab_project_mirror":             resourceGitlabProjectMirror(),
 			"gitlab_project_level_mr_approvals": resourceGitlabProjectLevelMRApprovals(),
 		},
+	}
 
-		ConfigureFunc: providerConfigure,
+	provider.ConfigureFunc = func(d *schema.ResourceData) (interface{}, error) {
+		return providerConfigure(provider, d)
 	}
```

Comment:
This is not needed

Reply:
Why do you think this is not needed? The `providerConfigure` function needs the `provider` object to get the Terraform version number. Wrapping the `providerConfigure` function allows us to do this. Is there another way to accomplish it?